Apps/phcanary/canary.json

```diff
@@ -0,0 +1,738 @@
+                    "required": true,
+                    "value_list": [
+                        "acknowledge",
+                        "unacknowledge"
+                    ],
+                    "default": "acknowledge",
+                    "order": 1
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 2,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.incident",
+                    "data_type": "string",
+                    "column_name": "Incident",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.incident_state",
+                    "data_type": "string",
+                    "column_name": "Incident State",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list incidents",
+            "identifier": "list_incidents",
+            "description": "Get list of existing Canary Incidents",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "node_id": {
+                    "description": "Get incidents from a specific node",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 1
+                },
+                "incident_state": {
+                    "description": "State of incidents to list",
+                    "data_type": "string",
+                    "required": true,
+                    "value_list": [
+                        "acknowledged",
+                        "unacknowledged"
+                    ],
+                    "default": "acknowledged",
+                    "order": 1
+                },
+                "tz": {
```

Comment:
`timezone` please also change in connector file

Reply:
Fixed. There is no reference in connector file.